### PR TITLE
docs: add instructions for testing

### PR DESCRIPTION
**Pull request by IgnisDa**
cc @alexanderniebuhr 

---
Commit messages:

```
docs(CONTRIBUTING.md): add instructions

Add instructions for testing on a real world
project.

feat: add testing project

build(settings.json): add settings for vscode live server
```

### CODE REVIEW

**Comment by high1** [comment]
### Test changes in a real project ### part uses yarn for dependency management, but the project and previous document parts use pnpm.

**Comment by antfu** [comment]
Why do we need this tho?

**Comment by IgnisDa** [comment]
@high1 Will look into that.

@antfu As a new contributor to windicss, I had trouble setting up a usable development environment. Asked @alexanderniebuhr about it on discord and he asked to make a PR if I wanted to.

**Comment by alexanderniebuhr** [comment]
@IgnisDa I think it is good to include instructions how to start the server.
Including an actual test project, is a bit over the top, I think :)

**Comment by antfu** [comment]
Ok, let's use `playground` for the folder name, and use `pnpm`'s workspace to share deps instead of `yarn`.

**Comment by IgnisDa** [comment]
`pnpm link` throws error, so that's why I used `yarn`. Mentioned that in the guide too.

File: pnpm-workspace.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- packages
+  - playground
```
Comment (hannoeru) [inline pnpm-workspace.yaml:2]:
```suggestion
packages:
  - playground
```

File: playground/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+!*
```
Comment (hannoeru) [inline playground/.gitignore:1]: why need this?

File: pnpm-workspace.yaml (hunk 1 above)
Comment (hannoeru) [reply pnpm-workspace.yaml:2]: @antfy looks like https://github.com/antfu/unplugin-vue2-script-setup/blob/main/pnpm-workspace.yaml is using the wrong syntax?

https://pnpm.io/pnpm-workspace_yaml

File: playground/.gitignore (hunk 2 above)
Comment (IgnisDa) [reply playground/.gitignore]: Yes that was not needed. Thanks for noticing!

File: .gitignore [hunk 3]
```diff
@@ -58,3 +58,4 @@ dist
 !test/extend.d.ts
 !test/plugin/typography/src/*.d.ts
 !test/plugin/typography/index.html
+!example/index.html
```
Comment (hannoeru) [inline .gitignore:61]: why add this?

Comment (IgnisDa) [reply .gitignore:61]: Since there is `*.html` in that file and that was getting ignored.

Comment (IgnisDa) [reply .gitignore]: Sorry I must have added that change before and forgotten to remove it. Thanks for noticing!

File: playground/windi.config.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  extract: {
+    include: ['**/*.{html,}'],
+    exclude: ['**/node_modules/**', '.git', 'dist', 'personal'],
```
Comment (hannoeru) [inline playground/windi.config.js:4]:
```suggestion
```

Windi CSS defaults ignore node_modules and .git folder, others are unnecessary.

File: playground/package.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  "version": "1.0.0",
+  "main": "index.js",
+  "author": "Diptesh Choudhuri",
+  "license": "MIT",
+  "devDependencies": {
+    "chokidar-cli": "^3.0.0"
+  },
+  "scripts": {
+    "start": "node ../dist/cli/index.js -to windi.css --dev -f windi.config.js --style",
+    "watch": "chokidar -d 1000 '../dist/**/*' -c \"npm run start\""
```
Comment (hannoeru) [inline playground/package.json:12]:
```suggestion
    "build": "node ../dist/cli/index.js -to windi.css --dev -f windi.config.js --style",
    "dev": "chokidar -d 1000 '../dist/**/*' -c \"npm run start\""
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -81,40 +82,67 @@ There are five steps to submit pull request:
...
+project.
+
+1. In the windicss root project directory, run:
+
+    ```bash
+    pnpm install
+    ```
+
+2. In the first terminal, run `pnpm run dev` in the windicss project directory, and in the
+   second one open the playground directory and run `pnpm run dev`.
```
Comment (hannoeru) [inline CONTRIBUTING.md:128]:
```suggestion
2. In the first terminal, run `pnpm dev` in the windicss project directory, and in the
   second one open the playground directory and run `pnpm dev`.
```
In pnpm you can run scripts without `run`. just to keep everything consistent in this document.

**Pull request merged**